berryc.1

```diff
@@ -67,6 +67,10 @@ Toggle the fullscreen status of the active window, don't change the size
 Move and resize the current window to fill the left half of the screen\. Respects TOP_GAP\.
 .
 .TP
+\fBomni\fR
```

Comment:
Perhaps I'm misunderstanding the intention of this function, but it doesn't behave the way I would expect it to. My thought was that, when activated, it would take the currently focused window with the user when they switch desktops. However, it seems instead to tag the currently selected window with the property and brings that window with the user to each desktop, even if they switch away and focus a different window. Is this the intention?

Certainly, one way to implement either of these behaviors is inside of `berry` itself by keeping track of a toggle for whether or a window or (windows) should be brought alone between workspace switches. Another idea, however, is to expand the functionality of `berryc` to allow for many commands to take a window id as an argument and apply that command to that window. For example, instead of something like:
```
berryc window_move 10 10
```
which can only ever move the focused window, we could do something like
```
berryc window_move 10 10 0x123456
```
which would move the specified window with id `0x123456`. This would be a nice for a plethora of reasons, but it would also allow us to move the behavior for something like omni-presence to the user. My rationale behind this is that there are so many ways one could want omni-presence to behave, and this would allow them to do that. Inside of `sxhkdrc` they could get the id of the focused window and send to the destination workspace anytime they want to switch desktops.

Reply:
i just wanted to have ability to put window to another monitor and have ability to make it sticky for all the workspaces when switching

for example, i can put terminal to another monitor, make it sticky, ssh to some machine, and start work there, while switching to another workspaces for googling smth on browser, or setting playlist on spotify

also i can put youtube to anither monitor and watch some videos on every desktop

this seems intuitive to me, tho

i don't pretend to have this functionality in berry, i just did this for my workflow, and share it with you, just in case i give some useful ideas